scripts/poe/lib/validate.js

```diff
@@ -54,7 +31,23 @@ const validDirectory = (fpath) => {
   return valid;
 };
 
+const validatePlaceHolders = (langs, dir) => {
+  const templateFile = `${dir}/messages-en.properties`;
+  langs.filter(lang => lang !== 'en').forEach(lang => {
+    const file = `${dir}/messages-${lang}.properties`;
+    const translations = extractPlaceholders(file);
+    const translationsTemplate = extractPlaceholders(templateFile);
+    translations.map((e1) => {
+      const e2 = translationsTemplate.find(t => t.key === e1.key);
+      if (e1.match.toString() !== e2.match.toString()) {
+        console.log(`\nWarning: messages-${lang}.properties: Translation key ${e1.key} on line ${e1.index + 1} has placeholders that do not match those of messages-en.properties`);
```

Comment:
I think we should fail validation if this happens otherwise people will just ignore it.